docs/img/deploy.svg

```diff
@@ -0,0 +1,282 @@
+			<v:cp v:nameU="ProcessNumber" v:lbl="Process Number" v:prompt="" v:type="2" v:format="" v:sortKey="" v:invis="false" v:ask="false" v:langID="1033" v:cal="0"/>
+			<v:cp v:nameU="Owner" v:lbl="Owner" v:prompt="" v:type="0" v:format="" v:sortKey="" v:invis="false" v:ask="false" v:langID="1033" v:cal="0"/>
+			<v:cp v:nameU="Function" v:lbl="Function" v:prompt="" v:type="0" v:format="" v:sortKey="" v:invis="false" v:ask="false" v:langID="1033" v:cal="0" v:val="VT4()"/>
+			<v:cp v:nameU="StartDate" v:lbl="Start Date" v:prompt="" v:type="5" v:format="" v:sortKey="" v:invis="false" v:ask="false" v:langID="1033" v:cal="0"/>
+			<v:cp v:nameU="EndDate" v:lbl="End Date" v:prompt="" v:type="5" v:format="" v:sortKey="" v:invis="false" v:ask="false" v:langID="1033" v:cal="0"/>
+			<v:cp v:nameU="Status" v:lbl="Status" v:prompt="" v:type="4" v:format=";Not Started;In Progress;Completed;Deferred;Waiting on Input" v:sortKey="" v:invis="false" v:ask="false" v:langID="1033" v:cal="0" v:val="VT4()"/>
+		</v:custProps>
+		<v:userDefs>
+			<v:ud v:nameU="visVersion" v:prompt="" v:val="VT0(15):26"/>
+			<v:ud v:nameU="DefaultWidth" v:prompt="" v:val="VT0(1):0"/>
+			<v:ud v:nameU="DefaultHeight" v:prompt="" v:val="VT0(0.75):0"/>
+			<v:ud v:nameU="ResizeTxtHeight" v:prompt="" v:val="VT0(0.75):0"/>
+		</v:userDefs>
+		<v:textBlock v:margins="rect(2,2,2,2)"/>
+		<v:textRect cx="38.8872" cy="285.109" width="77.78" height="37.8504"/>
+		<rect x="0" y="266.184" width="77.7744" height="37.8504" class="st3"/>
+		<text x="14.42" y="282.71" class="st4" v:langID="1033">
+			<v:paragraph v:horizAlign="1"/>
+			<v:tabList/>
+Cloud platform<v:newlineChar/>
+		<v:paragraph v:horizAlign="1"/>
+		<tspan x="22.15" dy="1.2em" class="st5">(Azure, ...)</tspan>
+	</text>
+</g>
+<g id="shape20-45" v:mID="20" v:groupContext="shape" v:layerMember="1" transform="translate(167.081,-75.8284)">
+	<title>Dynamic connector.20</title>
+	<desc>​Deploy</desc>
+	<v:textBlock v:margins="rect(4,4,4,4)"/>
+	<v:textRect cx="9" cy="322.898" width="60.01" height="22.4037"/>
+	<path d="M9 304.03 L9 335.72" class="st6"/>
+	<rect v:rectContext="textBkgnd" x="-8.08612" y="315.698" width="34.1721" height="14.4001" class="st10"/>
+	<text x="-0.674em" y="326.5" class="st9" v:langID="1033">
+		<v:paragraph v:horizAlign="1"/>
+		<v:tabList/>
+​Deploy</text>
+</g>
+<g id="shape21-52" v:mID="21" v:groupContext="shape" v:layerMember="1" transform="translate(78.6508,-85.7536)">
+	<title>Dynamic connector.21</title>
+	<desc>​Deploy</desc>
+	<v:textBlock v:margins="rect(4,4,4,4)"/>
+	<v:textRect cx="29.2716" cy="295.034" width="60.01" height="22.4037"/>
+	<path d="M0 295.03 L52.5 295.03" class="st6"/>
+	<rect v:rectContext="textBkgnd" x="12.1855" y="287.834" width="34.1721" height="14.4001" class="st10"/>
+	<text x="1.015em" y="298.63" class="st9" v:langID="1033">
+		<v:paragraph v:horizAlign="1"/>
+		<v:tabList/>
+​Deploy</text>
+</g>
+</g>
+</svg>
```

Comment:
Mm, this looks misaligned/wrong, now on both Firefox and eog. I guess the tool does not handle SVG right :(

Reply:
Much better now :) We could get rid of the forced white background on some text entries, but that's nitpicking enough

Reply:
The white background is used to make the text visible in dark mode. Unless the font color can change with system theme, it have to use two colors to make it visible safely.